### PR TITLE
[Milvus] Add labels to attu component

### PR DESCRIPTION
**Pull request by vladwa**
## What this PR does / why we need it:
The labels specified in the global values section are currently not being applied to the Attu deployment. This modification will ensure that the labels defined by users are incorporated into the Attu deployment

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

### CODE REVIEW

**Comment by yiwangdr** [comment]
@LoveEachDay @zwd1208 Could you take a look at this PR?

**Comment by LoveEachDay** [comment]
@vladwa Could you bump the chart version and app version?

**Comment by vladwa** [comment]
@LoveEachDay As suggested, I have bumped the chart and app version. Kindly review and approve.

File: charts/milvus/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 name: milvus
-appVersion: "2.3.1"
+appVersion: "2.3.2"
```
[the remark this answers is not in the document]
Comment (vladwa) [reply charts/milvus/Chart.yaml:3]: Incorporated above comment.

**Comment by vladwa** [comment]
@LoveEachDay Incorporated your comment. Please review and approve.

**Comment by LoveEachDay** [comment]
> @LoveEachDay Incorporated your comment. Please review and approve.

@vladwa You'd force push your commits. We don't see any code changes here. And also try to rebase your commits into one.

**Comment by vladwa** [comment]
@LoveEachDay Changes are reflecting now. Can you please review?

**Comment by LoveEachDay** [comment]
> @LoveEachDay Changes are reflecting now. Can you please review?

@vladwa For now your pr has 5 commits. You'd rebase your commits into one, try to use the `git rebase` command.

**Comment by LoveEachDay** [comment]
Everything is look good to me.

**Comment by vladwa** [comment]
@LoveEachDay When I executed the Git rebase command, it indicated that the branch is already up to date. Given you are repository manager,I believe you can squash commits and perform the merge. Could you kindly proceed with squashing and merging, please?

Here's the output of the Git rebase command for reference:

bash-3.2$ git rebase
Current branch add-labels-attu is up to date.

**Comment by LoveEachDay** [comment]
@vladwa Squash is not the same as rebase. Here's some instructions to follow:
1. git checkout <pr-branch>
2. git rebase -i master # open the interactive mode, try to replace first `pickup` with `reword`, then the following each `pickup` to `fixup`, then save with `:wq` which will prompt another window to git commit again
3. git push --force <remote> <pr-branch> # force push

**Pull request closed without merging**